### ISSUE TITLE
FR: Python wheels

### ISSUE THREAD

**Issue by ofek** [nmslib/hnswlib#603]
Hey there, great library! It would be nice if wheels were distributed to PyPI to avoid having a compiler as a dependency.

**Comment by yurymalkov** [comment]
Hi @ofek! Thank you! Can you help with doing that?

**Comment by fafayqa** [comment]
> Hey there, great library! It would be nice if wheels were distributed to PyPI to avoid having a compiler as a dependency.

EXACTLY! I was trying half a day today to search web exactly for such file but to no success. :-(
